File: APIProjetoFinal/AceleraPleno.API/Models/Mesa.cs
using AceleraPleno.API.Models.Enuns;

namespace AceleraPleno.API.Models
{
    public class Mesa : Entity
    {
        public string Descricao { get; set; }
        public int Lugares { get; set; }
        public Guid? ClienteId { get; set; }
        public virtual Cliente? Clientes { get; set; }
        public bool Ocupada { get; set; }
        public string Ambiente { get; set; }
        public StatusMesa StatusMesa { get; set; }
    }
}

File: FrontAngular/src/app/models/mesa.ts
import { Cliente } from 'src/app/models/cliente';
export interface Mesa {
  id: string;
  descricao: string;
  lugares: number;
  clienteId: string;
  clientes: Cliente;
  ocupada: boolean;
  ambiente: string;
  statusMesa: number;
  dataInclusao: Date;
  dataAlteracao: Date;
}
